libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/route/AnnotationsRefresher.kt

```diff
@@ -48,6 +48,16 @@ internal object AnnotationsRefresher {
             newAnnotation,
             startingLegGeometryIndex,
         ) { maxspeed() }
+        val freeFlowSpeed = mergeAnnotationProperty(
```

Comment:
Some changes are also required in `DirectionsRouteDiffProvider`.

Reply:
done